<launch>
  <node name="orb_slam2_bebop_mono" pkg="orb_slam2_ros"
      type="orb_slam2_ros_mono" args="
          $(find orb_slam2_ros)/orb_slam2/Vocabulary/ORBvoc.txt
          $(find orb_slam2_ros)/orb_slam2/config/bebop.yaml"
      output="screen">
       <remap from="/camera/image_raw" to="/bebop/image_raw" />

       <param name="publish_pointcloud" type="bool" value="true" />
       <param name="pointcloud_frame_id" type="string" value="map" />
       <param name="camera_frame_id" type="string" value="camera_link" />
       <param name="publish_scale" type="bool" value="true" />
  </node>
</launch>
